### DISCUSSION TITLE
naive un-optimised single-thread

### DISCUSSION THREAD

**Discussion by igouy** [hanabi1224/Programming-Language-Benchmarks#450] [Show and tell]
My impression is that [hanabi1224](https://github.com/hanabi1224) has increasingly wanted to make a comparison between programs that seem to be more or less the same?

_fwiw_ [Something like that](https://benchmarksgame-team.pages.debian.net/benchmarksgame/performance/comparable.html) has been added to the benchmarks game.

**Comment by hanabi1224** [comment]
Thanks for the info! Will take a look when I get a chance.
